openslides_backend/shared/patterns.py

```diff
@@ -11,101 +11,95 @@
 
 ID_PATTERN = re.compile(ID_REGEX)
 
+FullQualifiedId_REGEX = KEYSEPARATOR.join(
```

Comment:
Don't mix camel and full upper case, just use full upper for the full name since it's a constant

Reply:
done